python_modules/libraries/dagster-components/dagster_components_tests/integration_tests/validation_tests/test_cases.py

```diff
@@ -0,0 +1,71 @@
+from dataclasses import dataclass
+from pathlib import Path
+from typing import Callable, Optional
+
+
+@dataclass
+class ComponentValidationTestCase:
```

Comment:
this is a little overengineered, but I wanted to be able to run the same test cases for the CLI and for the raw loading error messages